Chapter_786_A.txt

```diff
@@ -0,0 +1,5 @@
+Learning about the main branch, we've commited two changes.
+Now that we've worked on one. We are proving our skills through our second one!
+Hope this worked.
+
+Now this is a story all about how my github story flipped upside down! Just kidding, we're making progress! Thank you Daniel.
```

Comment:
Now I'd like to take a minute, just sit right there, and I'll tell you how I am grateful that you came out for the workshop :)

Reply:
I still think about this class every time I open my IDE! Hope you're keeping well!!!